package-scripts/datadog-agent/postinst

```diff
@@ -31,7 +33,11 @@ if [ -f /etc/debian_version ] || [ "$LINUX_DISTRIBUTION" == "Debian" ] || [ "$LI
   set -e
   case "$1" in
       configure)
-          update-rc.d datadog-agent defaults
+          if which systemd >/dev/null 2>&1; then
+	    ln -s /lib/systemd/system/datadog-agent.service /etc/systemd/system/datadog-agent.service
```

Comment:
Yes we can. Weirdly though it may be, the update-rc.d call isn't equivalent to systemctl enable (no symlink exists under /etc/systemd/system after the call).